expenses-manager-frontend/src/main/frontend/src/styles.css

```diff
@@ -15,5 +15,13 @@ html, body {
     --secondary-color: #E9F0FF;
     --input-border-color: #646a77;
     --input-selected-border-color: #007bff;
-}
 
+    --food-restaurant-color: #46B1D9;
+    --car-color: #5844E3;
+    --subscriptions-color: #E047AB;
+    --entertainment-color: #F78486;
+    --coffee-color: #A1D12E;
+
+    --background-color: #ffffff;
```

Comment:
background color este foarte generic, daca chiar vrei sa adaugi culoarea alb in lista (poti folosi pur simplu white fara sa mai creezi variabila) as denumi --ligh-background-color, sau --white-background-color.